corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views/query.js

```diff
@@ -708,12 +723,14 @@ hqDefine("cloudcare/js/formplayer/menus/views/query", function () {
             if (self.dynamicSearchEnabled) {
                 self.updateSearchResults();
             }
+            this.updateSubmitButtonStatus(false);
```

Comment:
It'd be helpful to not have to look at the function definition to understand whether this is disabling or enabling. What if the function were `updateSubmitButtonEnabled`? Or `disableSubmitButton` / `enableSubmitButton` functions that called a helper?

Reply:
https://github.com/dimagi/commcare-hq/pull/34008#:~:text=rename%20updateSubmitButtonDisabled-,9721647,-Merge%20state